storage/skip_localtest.go

```diff
@@ -0,0 +1,12 @@
+package storage
```

Comment:
@mckim19 Copyright should be added on the top of the new file.

Reply:
Thank you for your review. 
I wiil add the copyright.